platform/linux-generic/arch/aarch64/odp_atomic.h

```diff
@@ -267,4 +267,9 @@ static inline __int128 __lockfree_load_16(__int128 *var, int mo)
 
 #endif
 
+static inline int atomic_is_lockfree_u128(void)
```

Comment:
Couldn't you use `odp_atomic_lock_free_u128()` instead? You just have to check the individual bits: load, cas

Reply:
Okay, I will use that and remove this API. Thanks!